src/main/java/frc/robot/Drive.java

```diff
@@ -120,12 +119,14 @@ public static void initializeDrive(){
         R2 = new CANSparkMax(2, MotorType.kBrushless);
         L1 = new CANSparkMax(3, MotorType.kBrushless); // init the motors
 		L2 = new CANSparkMax(4, MotorType.kBrushless);
-		Shifters = new DoubleSolenoid(ShiftersForward,ShiftersReverse);
 
-        R1.setIdleMode(IdleMode.kBrake);
-        L1.setIdleMode(IdleMode.kBrake);
+
+        // R1.setIdleMode(IdleMode.kBrake);
+        // L1.setIdleMode(IdleMode.kBrake);
         R2.follow(R1);
-        L2.follow(L1);
+		L2.follow(L1);
+		L1.setInverted(true);
+		L2.setInverted(true);
```

Comment:
Not sure if this is an issue in github or not, but looks like there might be funky indention issues here happening.